app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    author_id = params[:user_id]
+    @user = User.find(author_id)
+    @posts = @user.posts
+  end
 
-  def show; end
+  def show
+    id = params[:id]
+    @post = Post.find(id)
+  end
```

Comment:
- [ ] I am not able to see a user post when I check user posts I am getting an error please address that:
![post](https://user-images.githubusercontent.com/44745855/172699520-0c9aef28-1436-4ebb-be92-a6aefb0ba34a.png)

Reply:
- [ ] Remember that you should render the user's 3 most recent post